addons/web/static/src/core/autocomplete/autocomplete.js

```diff
@@ -27,6 +28,21 @@ export class AutoComplete extends Component {
         this.hotkeysToRemove = [];
 
         onWillUnmount(() => this.debouncedOnInput.cancel());
+
+        // position and size
+        const sourcesListRef = useRef("sourcesList");
+        useEffect(
+            () => {
+                if (sourcesListRef.el) {
+                    sourcesListRef.el.style.width =
```

Comment:
@mcm-odoo 
FYI: I think we always need to use the position hook.
But, I'm not sure about the horizontal size, ie, how it will behave when entries are big